keygen/src/keygen.rs

```diff
@@ -349,7 +388,47 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                         .validator(is_parsable::<usize>)
                         .default_value(&default_num_threads)
                         .help("Specify the number of grind threads"),
-                ),
+                )
+                .arg(
+                    Arg::with_name("mnemonic")
+                        .long("mnemonic")
+                        .alias("use-mnemonic")
+                        .help("Generate using a mnemonic key phrase"),
+                )
+                .arg(
+                    Arg::with_name("word_count")
+                        .requires_all(&["mnemonic"])
+                        .long("word-count")
+                        .possible_values(&["12", "15", "18", "21", "24"])
+                        .default_value("12")
+                        .value_name("NUMBER")
+                        .takes_value(true)
+                        .help("(requires mnemonic) Specify the number of words that will be present in the generated seed phrase"),
+                )
+                .arg(
+                    Arg::with_name("language")
+                        .requires_all(&["mnemonic"])
+                        .long("language")
+                        .possible_values(&["english", "chinese-simplified", "chinese-traditional", "japanese", "spanish", "korean", "french", "italian"])
+                        .default_value("english")
+                        .value_name("LANGUAGE")
+                        .takes_value(true)
+                        .help("(requires mnemonic) Specify the mnemonic lanaguage that will be present in the generated seed phrase"),
+                )
+                .arg(
+                    Arg::with_name("no_passphrase")
+                        .requires_all(&["mnemonic"])
+                        .long("no-bip39-passphrase")
+                        .alias("no-passphrase")
```

Comment:
This is copypasta from `new` right?  I think we can omit the alias since this is a new introduction